src/encoded/static/components/dataset.js

```diff
@@ -1939,6 +1948,12 @@ export const SeriesComponent = (props, reactContext) => {
                                 </div>
                             : null}
 
+
+                            <div data-test="diseasetermname">
+                                <dt>Disease{diseases && diseases.length !== 1 ? 's' : ''}</dt>
+                                <dd>{diseases && diseases.length > 0 ? diseases.join(', ') : 'Not reported'}</dd>
```

Comment:
I think technically `diseases` exists no matter what happens in the loops above, though it could have a zero length, so you don’t have to test for its existence here.